core/services/vrf/vrf_coordinator_solidity_crosscheck_test.go

```diff
@@ -28,9 +28,9 @@ import (
 	"github.com/smartcontractkit/chainlink/core/internal/gethwrappers/generated/solidity_vrf_request_id"
 )
 
-// coordinator represents the universe in which a randomness request occurs and
+// coordinatorUniverse represents the universe in which a randomness request occurs and
```

Comment:
Nitpick, but this rename could be done in a different PR, it's not needed for this change.

Reply:
Agreed. We had a failing CI test because of this though, `go vet shadow` - so I jumped on it